packages/rest-client-jsonapi/src/services/utils/resources/resources.test.ts

```diff
@@ -249,7 +261,11 @@ describe("convertToJsonApiRelationships", () => {
     }
 
     expect(
-      convertToJsonApiRelationships(sourceConfig, schema, relationships),
+      convertToJsonApiRelationships(
+        sourceConfig,
+        schemaMap.Article as any,
```

Comment:
the `schemaMap` defined higher up in the test is a shorthand that casts to `as Schema` because the tests here don't really care about the internals too much - just the `schema.type`, I updated this to cast `as Schema` rather than any